filesystem/ext2.ksy

```diff
@@ -167,8 +169,9 @@ types:
       inode_bitmap:
         pos: inode_bitmap_block * _root.bg1.super_block.block_size
         size: 1024
-      # http://www.nongnu.org/ext2-doc/ext2.html#INODE-TABLE
-      # http://web.archive.org/web/20161114202411/http://www.virtualblueness.net/Ext2fs-overview/Ext2fs-overview-0.1-10.html
+        doc-ref:
+        - http://www.nongnu.org/ext2-doc/ext2.html#INODE-TABLE
```

Comment:
wrong indent.

Reply:
Wouldn't a doc-ref a level down cause it to count as an enum? OR should I go a level higher?

Reply:
@Icekhaos  See [KSY Style Guide > 1. General formatting](https://doc.kaitai.io/ksy_style_guide.html#general):

> Formatting of sequences MUST include an indent, i.e.:
>
> <table>
> <thead>
> <tr>
> <th align=left>✔️ Good</th>
> <th align=left>❌ Bad</th>
> </tr>
> </thead>
> <tbody>
> <tr>
> <td>
> 
> ```yaml
> imports:
>   - foo
>   - bar
> ```
>
> </td>
> <td>
> 
> ```yaml
> imports:
> - foo
> - bar
> ```
>
> </td>
> </tr>
> </tbody>
> </table>

Reply:
Oh, okay.